source/developers-guide/vagrant-phpstorm/index.md

```diff
@@ -146,15 +146,30 @@ Input fields:
 After you get the `"BUILD SUCCESSFUL"` message you can run the full build command.
 
     $ ant build-unit
-  
+
+### Step 3
```

Comment:
Is that still necessary? I've never used the `RewriteBase` for shopware installations on various setups. I pretty sure it works well on the vagrant machine without that step.